business_requirement_deliverable_report/__openerp__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement Deliverable Report Docx Module',
+    'version': '9.0.1.0.3',
+    'category': 'report',
+    'depends': [
+        'report_docx',
```

Comment:
True: we need to create a PR for base-report.
Nevertheless we will probably rewrite the reports in qweb due to the limitation of this engine (no html support which is annoying as all main fields are html content)